include/debug/cpu_load.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef __CPU_LOAD_H
+#define __CPU_LOAD_H
+
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup cpu_load CPU load
+ * @brief Module for measuring CPU load.
+ *
+ * @{
+ */
+
+/** @brief Initialize CPU load measurement module.
+ *
+ * During initialization TIMER driver and PPI channels are allocated.
+ *
+ * @return 0 on success, error code otherwise.
+ */
+int cpu_load_init(void);
+
+/** @brief Reset measurement. */
+void cpu_load_reset(void);
+
+/** @brief Get CPU load.
+ *
+ * Load is represented in 0,001% units where 100000 value represents 100% load.
```

Comment:
```suggestion
* The CPU load is represented in 0,001% units where a 100000 value represents 100% load.
```
Can you add an example value here? Where do you get these values? In the shell?